docs/_source/practical_guides/create_dataset.md

```diff
@@ -115,6 +115,28 @@ The following arguments apply to specific metadata types:
 You can also define metadata properties after the dataset has been configured or add them to an existing dataset in Argilla. To do that use the `add_metadata_property` method as explained [here](/practical_guides/create_dataset.md).
 ```
 
+##### Define `vectors`
+
+To use the similarity search in the UI and the Python SDK, you will need to configure vector settings. These are defined using the SDK as a list of up to 5 vectors. They have the following arguments:
+
+- `name`: The name of the vector, as it will appear in the records.
+- `dimensions`: The dimensions of the vectors used in this setting.
+- `title` (optional): A name for the vector to display in the UI for better readability.
+
+```python
+vector_settings = [
+    rg.VectorSettings(
+        name="my_vector",
+        dimensions=768
+    ),
+    rg.VectorSettings(
+        name="my_other_vector",
+        title="Another Vector", # optional
+        dimensions=768
+    )
+]
+```
+
```

Comment:
Nothing about how to configure them into a dataset?

Reply:
That appears below with the rest of the settings. If you think it's worth considering changing the layout I can tackle that in a separate issue.